### DISCUSSION TITLE
Changing the "Sort by..." label in the sort dropdown to French

### DISCUSSION THREAD

**Discussion by sinc13** [microsoft-search/pnp-modern-search#3609] [Q&A]
Is it possible to edit or customize the default "Sort by..." label in the sort dropdown? We would like that to appear in French for French users or on French pages.

Thanks

**Comment by kasperbolarsen** [comment]
I would actually expect it to be translated already
Are you refering to this one? ( in the list, cards and slide Layouts) 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/b6219994-1caa-47cd-aea7-834ae2cd1744)

**Comment by kasperbolarsen** [comment]
In the source code:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/5dfe0142-b822-483d-99a8-50c2b07f6fcc)

so the text in the Sort by box should follow the rest of the page

**Comment by sinc13** [comment]
Yes that's the one. Unfortunately the French label does not appear for a French user. As you can see in the screenshot, all the other elements on the page such as the date interval values and the publishing toolbar are in French, but the Sort by remains in English.

![SearchDropdown](https://github.com/microsoft-search/pnp-modern-search/assets/29189720/bfd01a19-49e9-42b3-9003-43e4d6a47312)

**Reply by kasperbolarsen** [reply]
I will see if I can recreate this problem in another language,  and change this to an issue/bug

**Reply by wobba** [reply]
> Yes that's the one. Unfortunately the French label does not appear for a French user. As you can see in the screenshot, all the other elements on the page such as the date interval values and the publishing toolbar are in French, but the Sort by remains in English.
> 

1. What's the default language on your web page and does it allow other languages?
2. Are other OOB text in French?
3. What's the browser locale you use?

**Comment by kasperbolarsen** [comment]
It seems like this field isn't translated. I tried in my own language and the English text was displayed. I have converted this to an issue
